Dockerfiles/python/.devcontainer/Dockerfile

```diff
@@ -52,13 +52,16 @@ ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app
 ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-python-template/main/app/requirements-links.txt /requirements/
 ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-python-template/main/app/requirements-app.txt /requirements/
 ADD --chown=4000 https://raw.githubusercontent.com/eclipse-velocitas/vehicle-app-python-template/main/app/tests/requirements-tests.txt /requirements/
+COPY --chown=4000 ./common/requirements-common.txt /requirements/
```

Comment:
Actually this copy is not really needed. The `requirements-common.txt` is already available in the container via the COPY in line 39 `/scripts/requirements-common.txt`. Instead of copying it again, we may change the name of the copy target from `/scripts` to `/setup` or `/common`. Same also applies for the Cpp image.